packages/services/f-braze-adapter/src/services/utils/isCardCurrentlyActive.js

```diff
@@ -28,12 +28,14 @@ export function isCardActiveBasedOnTime (displayTimes) {
  * Can be used in isolation or as part of Array.filter
  * @param card {object}
  * @param card.displayTimes {object} - Display times
- * @param card.brand {string} - Identifier string for relevant brand
+ * @param card.isVisible {string} - Identifier string for relevant brand
```

Comment:
still references brand in comment, my bad for leaving it in here.

Reply:
Good spot, will update!